.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "assets/scss/styleguide"]
 	path = assets/scss/styleguide
 	url = https://github.com/greenpeace/planet4-styleguide.git
+[submodule "assets/src/scss/styleguide"]
+	path = assets/src/scss/styleguide
+	url = https://github.com/greenpeace/planet4-styleguide
```

Comment:
Let's remove the old submodule deceleration and keep just the new one. I think the url should end with `.git`, but maybe I'm wrong.